R/GraphClasses.R

```diff
@@ -128,38 +128,38 @@ AbstractGraph <- R6::R6Class(
     classname = "AbstractGraph"
     , public = list(
         initialize = function(nodes, edges) {
-
+            
```

Comment:
Can you please go into Options > Code > Saving and enable these two settings?

- Ensure that source files end with newline
- Strip trailing horizontal whitespace when saving

They will automatically format your code when you save to remove whitespace like that. We do this so that our code is consistent and we don't end up with extraneous diffs like this. [Apologies that this isn't in the instructions anywhere yet!](https://github.com/uptake/pkgnet/issues/264)

Reply:
I have created an updated commit after making changes as per above. But not sure how to tell if I did it correctly? If not, how do you suggest I do it. Should I simply copy a file from latest commit in master and redo the changes.

Reply:
This looks okay now, thanks for fixing.